src/lib/utils/metadata/NftDriverMetadataManager.ts

```diff
@@ -17,6 +18,29 @@ export default class NftDriverMetadataManager extends MetadataManagerBase<
     super(nftDriverAccountMetadataParser, nftDriver);
   }
 
+  /**
+   * Ensures that ecosystem metadata is not fetched. We shouldn't be handling it at this point.
```

Comment:
Minor suggestion: "fetches NFTDriver metadata but ensures that...." you get the idea.